content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,142 @@
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send text or binary data over an active connection to a peer.
+  Data channels can be [ordered or unordered](https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/ordered),
+  where ordered channels guarantee the message delivery order (like running on TCP), and unordered channels increase
+  speed but deliver messages out of order (like UDP.)
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/))to connect to nodes that live behind
+  NATs and firewalls.
+
+- [Security](../secure-comm/overview): WebRTC includes built-in security features, and
+  connections are always encrypted by using [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security)
+  or [SRTP](https://en.wikipedia.org/wiki/Secure_Real-time_Transport_Protocol).
+
+WebRTC includes several APIs to help create a secure connection
+over the web. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate using a peer-to-peer protocol. WebRTC enables
+a node to send and receive streams that include media content and arbitrary binary data
+through the `DataChannel`.
+
+## WebRTC in libp2p
+
+libp2p WebRTC enables browsers to connect to public server nodes (and eventually,
+browsers to connect to other browsers) without those endpoints providing a TLS
+certificate within the browser's trustchain.
+
+In libp2p:
+
+- WebRTC multiaddresses are composed of a standard UDP multiaddr,
+  followed by `webrtc` and the `multihash` of the certificate that
+  the node uses, as such:
+  `/ip4/1.2.3.4/udp/1234/webrtc/certhash/<hash>/p2p/<peer-id>`;
+- WebRTC offers security via TLS 1.2. Peers still need to authenticate remote peers
+  by their libp2p identity.
+- the TLS certificate fingerprint in `/certhash` is a multibase encoded multihash;
+- WebRTC can support UDP and TCP, but implementations must always support UDP.
+
+### Browser-to-Server
+
+A browser can connect to a server node without needing a trusted TLS
+certificate. View [this example](https://github.com/libp2p/specs/blob/master/webrtc/README.md#browser-to-public-server) on browser-to-server connection establishment in the technical
+specification.
+
+In general, a server will act as an [ICE Lite](https://www.rfc-editor.org/rfc/rfc5245)
+agent and binds to a UDP port waiting for incoming STUN and SCTP packets and multiplexes
+based on source IP and source port. It multiplexes based on a source IP and source port.
+
+Once a browser discovers a server's multiaddr, it instantiates an `RTCPeerConnection`.
```

Comment:
This should be phrased differently. We don't connect once we discover the address, but once we want to do something!

So: When a browser wants to establish a WebRTC connection to a server...